src/app/services/authorization/authorization.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { UserModel } from '../../core/user.model';
+import { Router } from '@angular/router';
+
+@Injectable()
+export class AuthorizationService {
+  private storage = new Map();
```

Comment:
Also for now you may use LocalStorage to save your credentials even after page reloading